### ISSUE TITLE
Maestro build failing on for focal

### ISSUE THREAD

**Issue by kunal11191** [PelionIoT/distro-pelion-edge#145]
Maestro build job is failing for focal with following command sequence 
```
./build-env/bin/docker-ubuntu-focal-create.sh
./build-env/bin/docker-run.sh  pelion-focal-source
./build-env/bin/pelion-build-all.sh --deps --install
./maestro/deb/build.sh --install --build --source --arch=amd64
```
Error message
```
x86_64-linux-gnu-g++: error: deps/build/lib/libuv.a: No such file or directory
make[2]: *** [Makefile:83: libgrease.so.1] Error 1
make[2]: Leaving directory '/pelion-build/build/tmp-build/maestro/maestro-0.0.1/go-workspace/src/github.com/armPelionEdge/maestro/vendor/github.com/armPelionEdge/greasego/deps/src/greaseLib'
make[1]: *** [debian/rules:10: override_dh_auto_build] Error 2
make[1]: Leaving directory '/pelion-build/build/tmp-build/maestro/maestro-0.0.1'
make: *** [debian/rules:7: build] Error 2
dpkg-buildpackage: error: debian/rules build subprocess returned exit status 2
script returned exit code 2
```
Note: Am able to build other components on focal and maestro works fine on other distribution.

**Comment by kbgl** [comment]
Issue should be fixed in #150

**Comment by kunal11191** [comment]
closing this as jobs are passing now
